README.md

```diff
@@ -1466,11 +1466,40 @@ See the official [MQTT authentication & authorization guide](https://www.loraser
 
 ### Docker
 
-This project provides example Dockerfiles for building a Docker container that contains `mosquitto` and the `mosquitto-go-auth` plug-in.
+#### Support and issues
+Please be aware that, since Docker isn't actively used by the maintainer of this project, support for issues regarding Docker, the provided images and building Docker images in very limited. 
```

Comment:
```suggestion
Please be aware that, since Docker isn't actively used by the maintainer of this project, support for issues regarding Docker, the provided images and building Docker images is very limited and usually driven by other contributors.
```